app/src/main/java/org/mozilla/fenix/components/toolbar/DefaultToolbarMenu.kt

```diff
@@ -411,12 +412,23 @@ open class DefaultToolbarMenu(
         context.getString(R.string.sync_menu_sign_in)
     }
 
-    val syncedTabs = BrowserMenuImageText(
-        syncItemTitle,
-        R.drawable.ic_synced_tabs,
-        primaryTextColor()
-    ) {
-        onItemTapped.invoke(ToolbarMenu.Item.SyncedTabs)
+    val syncTabsOrSignInItem = if(tabsTrayRewrite) {
+        BrowserMenuImageText(
+            context.getString(R.string.synced_tabs),
+            R.drawable.ic_synced_tabs,
+            primaryTextColor()
+        ) {
+            onItemTapped.invoke(ToolbarMenu.Item.SyncedTabs)
+        }
+    } else {
+        BrowserMenuImageText(
+            syncItemTitle,
+            R.drawable.ic_synced_tabs,
+            primaryTextColor()
+        ) {
+            onItemTapped.invoke(ToolbarMenu.Item.SyncAccount)
+        }
+
     }
```

Comment:
If we're trying to coordinate these two features landing at different times and avoiding the time where no 'sync tabs' show up anywhere, then we should show sign in + tabs or just sign in.

In which case, do not use the `tabsTrayRewrite` flag to switch the two, use the flag to show 'Sync tabs' only.

Reply:
Victoria's designs want synced tabs OR sync sign in. We will never show both. So as soon as the synced tabs move to the tabs tray we will start showing the sync sign in instead :)

Reply:
My comment wasn't about altering the final designs, but it looks like the 3-dot menu and synced tabs in the tray will not perfectly in the same weeks, so there will be a time in which nightly users will have the new three-dot menu and no new tabs tray so they won't be able to access their synced tabs. 🙂

Reply:
Ahh I see what you mean. I have them switched.

Reply:
Yeah, that works too!